src/components/form/dt-text/dt-text.js

```diff
@@ -54,6 +55,12 @@ export class DtTextField extends DtFormBase {
     };
   }
 
+  constructor(props) {
+    super(props);
+    this.type = 'text';
```

Comment:
The constructor initialization of props actually gets overwritten, which is a little deceiving, so this doesn't do anything. It can be removed.

Reply:
Done!